src/style.css

```diff
@@ -1 +1,36 @@
-/* add styles here */
+* {
```

Comment:
Using the universal selector (`*`) can impact performance as it applies styles to every single element on the page. It's generally better to apply `box-sizing: border-box;` to specific elements or a group of elements that require it. This is a performance issue according to the checklist.